stackslib/src/chainstate/nakamoto/tests/node.rs

```diff
@@ -1056,6 +983,22 @@ impl<'a> TestPeer<'a> {
         let mut stacks_node = self.stacks_node.take().unwrap();
         let sortdb = self.sortdb.take().unwrap();
 
+        let burn_height = self
```

Comment:
Can this 6-line-long chain of references be turned into a method?

Reply:
Actually, I think lines 986 through 1001 should be their own method, since you have it listed out twice.

Reply:
Good idea. Refactored in 0fdaf62fb.